File: NCore/Player/Inventory.cs
using System.Collections.Generic;
using System.Threading.Tasks;

namespace NCore
{
    public class Inventory
    {
        public List<ItemStack> Items;
        public NetworkPlayer Owner;

        public async Task AddItem(ItemStack arg0)
        {
            Items.Add(arg0);
        }

        public Inventory(NetworkPlayer arg0)
        {
            Owner = arg0;
            Items = new List<ItemStack>();
        }

        public int CountOf(Material m)
        {
            foreach (var i in Items)
            {
                if (i.Type == m)
                {
                    return i.Count;
                }
            }

            return -1;
        }
    }
}

File: NetcraftServer/Player/Inventory.vb
Public Class Inventory
    Public Items As List(Of ItemStack)
    Public Owner As NetworkPlayer
    Public Sub AddItem(arg0 As ItemStack)
        Items.Add(arg0)
    End Sub
    Sub New(arg0 As NetworkPlayer)
        Owner = arg0
        Items = New List(Of ItemStack)
    End Sub
    Function CountOf(m As Material) As Integer
        For Each i In Items
            If i.Type = m Then
                Return i.Count
            End If
        Next
        Return -1
    End Function
End Class
